sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/test/scala/com/azure/cosmos/spark/CosmosCatalogSpec.scala

```diff
@@ -91,11 +91,126 @@ class CosmosCatalogSpec extends IntegrationSpec with CosmosClient {
+
+    spark.sql(s"CREATE DATABASE testCatalog.${databaseName};")
+    spark.sql(s"CREATE TABLE testCatalog.${databaseName}.${containerName} (word STRING, number INT) using cosmos.items " +
+      s"TBLPROPERTIES(partitionKeyPath = '/mypk', manualThroughput = '1100', indexingPolicy = 'AllProperties')")
+
+    val containerProperties = cosmosClient.getDatabase(databaseName).getContainer(containerName).read().block().getProperties
+    containerProperties.getPartitionKeyDefinition.getPaths.asScala.toArray should equal(Array("/mypk"))
+    containerProperties
+      .getIndexingPolicy
+      .getIncludedPaths
+      .asScala
+      .map(p => p.getPath)
+      .toArray should equal(Array("/*"))
+    containerProperties
+      .getIndexingPolicy
+      .getExcludedPaths
+      .asScala
+      .map(p => p.getPath)
+      .toArray should equal(Array(raw"""/"_etag"/?"""))
+
+    // validate throughput
+    val throughput = cosmosClient.getDatabase(databaseName).getContainer(containerName).readThroughput().block().getProperties
+    throughput.getManualThroughput shouldEqual 1100
+  }
+
+  it can "create a table with well known indexing policy 'OnlySystemProperties'" taggedAs (RequiresCosmosEndpoint) in {
+    val databaseName = getAutoCleanableDatabaseName()
+    val containerName = RandomStringUtils.randomAlphabetic(6).toLowerCase + System.currentTimeMillis()
+
+    spark.sql(s"CREATE DATABASE testCatalog.${databaseName};")
+    spark.sql(s"CREATE TABLE testCatalog.${databaseName}.${containerName} (word STRING, number INT) using cosmos.items " +
+      s"TBLPROPERTIES(partitionKeyPath = '/mypk', manualThroughput = '1100', indexingPolicy = 'ONLYSystemproperties')")
+
+    val containerProperties = cosmosClient.getDatabase(databaseName).getContainer(containerName).read().block().getProperties
+    containerProperties.getPartitionKeyDefinition.getPaths.toArray should equal(Array("/mypk"))
+    containerProperties
+      .getIndexingPolicy
+      .getIncludedPaths
+      .asScala.map(p => p.getPath)
+      .toArray
+      .size shouldEqual 0
+    containerProperties
+      .getIndexingPolicy
+      .getExcludedPaths
+      .asScala
+      .map(p => p.getPath)
+      .toArray should equal(Array("/*", raw"""/"_etag"/?"""))
+
+    // validate throughput
+    val throughput = cosmosClient.getDatabase(databaseName).getContainer(containerName).readThroughput().block().getProperties
+    throughput.getManualThroughput shouldEqual 1100
+  }
+
+  it can "create a table with custom indexing policy" taggedAs (RequiresCosmosEndpoint) in {
+    val databaseName = getAutoCleanableDatabaseName()
+    val containerName = RandomStringUtils.randomAlphabetic(6).toLowerCase + System.currentTimeMillis()
 
+    val indexPolicyJson = raw"""{"indexingMode":"consistent","automatic":true,"includedPaths":""" +
+      raw"""[{"path":"\/helloWorld\/?"},{"path":"\/mypk\/?"}],"excludedPaths":[{"path":"\/*"}]}"""
```

Comment:
please also consider adding a pyspark sample.

it would be good to see how the escaping looks like in pyspark.

Reply:
ACK - tracked here: https://github.com/orgs/Azure/projects/64#card-57093087